using_openshift/cli.adoc

```diff
@@ -242,13 +242,17 @@ The syntax to describe a deployment configuration in human-readable form is:
 ****
 
 === Deployment Rollbacks
-Use the `osc rollback` command to revert part of an application back to a previous deployment.
 
-When you run this command your deployment configuration will be updated to match the provided deployment. By default, only the pod and container configuration will be changed, and scaling or trigger settings are left as-is. Note that environment variables and volumes are included in rollbacks; if you've recently updated security credentials in your environment, then your previous deployment may not have the correct values.
+Rollbacks revert an application back to a previous deployment. Note that environment variables and volumes are included in rollbacks, so there are a variety of considerations to make when deciding whether a rollback is viable, such as:
 
-If you would like to review the outcome of the rollback, then use `--dry-run` to print a human-readable representation of the updated deployment configuration instead of executing the rollback. This is useful if you're not quite sure what the outcome will be.
+* If security credentials have been recently updated, the previous deployment may not have the correct values.
+* If the previous deployment used a custom strategy, which is no longer available or usable, the deployment may not be deployed correctly.
 
-The syntax for `rollback` is:
+By default, only the pod and container configuration will be changed and scaling or trigger settings will be left as-is.
+
+Using the `-d`, or `--dry-run` option prints a human-readable representation of the updated deployment configuration instead of executing the rollback. This is useful for inspecting a desired outcome before an actual rollback.
+
+Use the `osc rollback` command to revert part of an application back to a previous deployment. The syntax for `rollback` is:
 
 ****
 `openshift cli rollback [replaceable]#<deployment># [replaceable]#[options]#`
```

Comment:
@bfallonf Re: the `openshift cli` vs `osc` discussion, we should probably use `osc` consistently in examples in these `using_openshift` topics instead of `openshift cli` (their relationship is discussed at the top of this file).